pkg/capabilities/capabilities.go

```diff
@@ -71,6 +71,7 @@ type RequestMetadata struct {
 type RegistrationMetadata struct {
 	WorkflowID    string
 	WorkflowOwner string
+	ReferenceID   string
```

Comment:
reference to what? inline comment?

Reply:
good point, I was tempted to update to StepReferenceID, but ReferenceID is already used elsewhere  for the same so went for consistency

Reply:
comment/rename?  I'll go comment unless objections